bindings/go/Makefile.am

```diff
@@ -14,12 +14,10 @@ export CGO_CFLAGS=$(CGOCFLAGS)
 export CGO_LDFLAGS=$(CGOLDFLAGS)
 
 build:
-	$(GO) env
-	$(GO) build
+	$(AM_V_at) $(GO) build $(abs_top_builddir)/bindings/go
```

Comment:
1. remove space after $(AM_V_at)
2. are you sure it will work? current directory is already `$(abs_top_builddir)/bindings/go`
 probably better add OOS go build test to CI?